### ISSUE TITLE
Null safety 

### ISSUE THREAD

**Issue by hatemragab** [zeeshux7860/ImageEditorPro#21]
is there plan to migrate to null safety

**Comment by yansarorp** [comment]
Yes, this package it's great but it's pretty outdated, would be great if you could update it to null safety

**Comment by bensal-kb** [comment]
Yes, it also conflicts with other packages
